app/src/main/res/menu/feedlist.xml

```diff
@@ -57,4 +57,11 @@
         custom:showAsAction="collapseActionView">
     </item>
 
-</menu>
\ No newline at end of file
+    <item
```

Comment:
Thanks for working on this! Could you move this up, please? I think "delete" should be the last item. Maybe it could even be above "share"

Reply:
I would say Share should be above Remove from inbox.

Reply:
Changed, thanks for the feedback